doc/governance.rst

```diff
@@ -20,95 +20,107 @@ the project community.
 Roles And Responsibilities
 ==========================
 
+We distinguish between contributors, core contributors, and the technical
+committee. A key distinction between them is their voting rights: contributors
+have no voting rights, whereas the other two groups all have voting rights,
+as well as permissions to the tools relevant to their roles.
+
 Contributors
 ------------
 
 Contributors are community members who contribute in concrete ways to the
 project. Anyone can become a contributor, and contributions can take many forms
 – not only code – as detailed in the :ref:`contributors guide <contributing>`.
+There is no process to become a contributor: once somebody contributes to the
+project in any way, they are a contributor.
+
+Core Contributors
+-----------------
+
+All core contributor members have the same voting rights and right to propose
+new members to any of the roles listed below. In order to join one of the roles
+bellow. Their membership is represented as being an organization member on the
+scikit-learn `GitHub organization
+<https://github.com/orgs/scikit-learn/people>`_.
+
+They are also more than welcome to join our `monthly core contributor meetings
```

Comment:
nit: can we re-write it as "All core contributors are welcome at the monthly core contributor's meeting.". Something about "more than welcome" triggers me.

Reply:
Aren't the monthly meetings open to anyone? Would it be better to call those monthly meetings just "sklearn monthly meetings"? or "contributor monthly meetings"

Reply:
So the monthly meetings are very distinct from the triage meetings, office hours, or other meetings we have every now and then. They are meetings where everybody is welcome, but we implicitly have had the expectation that a good part of the core team comes there, and we usually talk about things which are easier to talk about when that team is there and can have a sync face to face conversation. However, we also have core developers who pretty much never show up, and we haven't cared about that. That's why I worded it as "more than welcome".

How would you write that?

Reply:
I'd drop the "more than". For me that makes it more of a positive statement compared to the current wording.

Reply:
There is a monthly meeting (next on 27 March) which is open to everyone. But I don't know if there is also a private "core team only" meeting @adam2392